static/stripe.html

```diff
@@ -191,7 +194,7 @@ <h4 class="col s12">Thank you for your donation!</h4>
 
     // Keep track of the current PaymentIntent so we know whether or not to create a new one;
```

Comment:
This comment needs correcting - it's not technically a PaymentIntent since it's what Impact API responds with, not what Stripe responds with. All it contains is some basic info like `amount` and `premium` along with the `client_secret` _for the PaymentIntent_